README.md

```diff
@@ -7,73 +7,82 @@
+   - Ex. you can add custom configuration options or change parameters passed to child PHPUnit processes
+- Cloud services like [Sauce Labs](https://saucelabs.com/), [BrowserStack](https://www.browserstack.com/) or [TestingBot](https://testingbot.com/) are fully integrated giving you a chance to run tests with less setup and without your own infrastructure
+- Steward is field tested - we use it daily in [our company](https://www.lmc.eu/english) to maintain the quality of our products thanks to hundreds of test-cases
+   - the library itself is also extensively covered by unit tests
 - Steward is built on solid foundations: [WebDriver](http://www.w3.org/TR/webdriver/) is W3C draft standard for browser automation,
 [php-webdriver](https://github.com/facebook/php-webdriver) is the most used and developed Selenium language binding for PHP,
-[PHPUnit](https://phpunit.de/) is well known and widely used testing framework and
-[Symfony Console](http://symfony.com/doc/current/components/console.html) is industry standard for PHP CLI applications.
+[PHPUnit](https://phpunit.de/) is a well known and widely used testing framework, and
+[Symfony Console](http://symfony.com/doc/current/components/console.html) is industry standard for PHP CLI applications
 
 ## Example usage
 To see how to use and extend Steward, have a look at our [example project](https://github.com/lmc-eu/steward-example).
 
 ## Changelog
-For latest changes see [CHANGELOG.md](CHANGELOG.md) file. We follow [Semantic Versioning](http://semver.org/).
+For the latest changes see the [CHANGELOG.md](CHANGELOG.md) file. We follow [Semantic Versioning](http://semver.org/).
 
 ## Getting started
 ### 1. Install Steward
 For most cases we recommend having functional tests in the same repository as your application but in a separate folder.
-So let's suggest we put them in `selenium-tests/` directory.
+We suggest putting them in a `selenium-tests/` directory.
 
-**In this directory** then simply install Steward:
+**In this directory**, simply install Steward with the following command:
 
 ```sh
 $ composer require lmc/steward
 ```
 
-Note you will need to have [Composer](https://getcomposer.org/) installed to do this.
+**Note:** you will need to have [Composer](https://getcomposer.org/) installed to do this.
 
-### 2. Download Selenium server and browser driver
-You can download and run Selenium standalone server and the browser locally right on your computer.
-Another possibility is to [start Selenium server + browser inside Docker container][wiki-docker].
-To run Selenium locally:
+### 2. Download Selenium Server and browser drivers
+You can download and run Selenium Standalone Server with the test browser opening locally right on your computer.
+Another possibility is to [start Selenium Server and test browser inside a Docker container][wiki-docker].
 
-#### Get Selenium standalone server
+####To run Selenium locally:
```

Comment:
This is not properly formatted. And I think this header is unnecessary - better would IMHO be to reword previous sentences to something like `This step only applies if you want to download and run Selenium Standalone Server with the test browser locally right on your computer. Another possibility is to [start Selenium Server and test browser inside a Docker container][wiki-docker].`.

What do you think, would it be understandable?

Reply:
I agree with this. I'll make the change.